docs/installing/README.md

```diff
@@ -12,13 +12,31 @@ There are so many platforms out there that they didn't fit into the main
 README.md file, so here are some installation instructions for other platforms
 from the community. Is yours not here? Please do add it here if you figure it
 out!
+
+## [Chocolatey](https://chocolatey.org)
+
+### Prerequisites
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
This is a chocolatey install instruction right?

I'd prefer not to have that directly in our install instructions, since that means that if the instructions ever change in the future, we'd need to modify our code to keep up. I'd much rather prefer a link to https://chocolatey.org/install and a note saying that you need to make sure chocolatey is installed.

Reply:
I've made the necessary changes @chipbuster. Do have a look! Thanks.